packages/fluentui/react-northstar-prototypes/src/prototypes/Prototypes.tsx

```diff
@@ -10,27 +12,27 @@ interface ComponentPrototypeProps extends PrototypeSectionProps {
   description?: React.ReactNode;
 }
 
-export const PrototypeSection: React.FunctionComponent<ComponentPrototypeProps> = props => {
-  const { title, children, styles, ...rest } = props;
-  return (
-    <Box styles={{ margin: '20px', ...styles }} {...rest}>
-      {title && <Header as="h1">{title}</Header>}
-      {children}
-    </Box>
-  );
-};
+export const PrototypeSection: React.FunctionComponent<PrototypeSectionProps> = ({ children, styles, title }) => (
+  <Box styles={{ margin: '20px', ...styles }}>
+    <DocumentTitle title={`Fluent UI - ${title || 'Prototype'}`} />
+    <Header as="h1">{title}</Header>
+    {children}
+  </Box>
+);
 
-export const ComponentPrototype: React.FunctionComponent<ComponentPrototypeProps> = props => {
-  const { description, title: header, children, styles, ...rest } = props;
-  return (
-    <Box styles={{ marginTop: '20px', ...styles }} {...rest}>
-      {(header || description) && (
-        <Segment>
-          {header && <Header as="h3">{header}</Header>}
-          {description && <p>{description}</p>}
-        </Segment>
-      )}
-      <Segment>{children}</Segment>
-    </Box>
-  );
-};
+export const ComponentPrototype: React.FunctionComponent<ComponentPrototypeProps> = ({
+  children,
+  description,
+  styles,
+  title,
+}) => (
+  <Box styles={{ marginTop: '20px', ...styles }}>
```

Comment:
are you sure there is no more need for `...rest` spreading ? to help avoid unintended consequences I would say just keep the original signature

Reply:
If the typing is correct, `rest` would always be empty as all properties are already desctructured:
![empty `rest`](https://user-images.githubusercontent.com/2564094/121961665-8f300780-cd1c-11eb-9d38-d9d9e652a1dc.png)

Reply:
Thx for checking, although northstar doesn't use `strict` typings and there are a fair amount of `any` traps lying around the current codebase.

In any case I don't think this blocks the PR, it looks good otherwise